src/utils/gauges.ts

```diff
@@ -15,10 +24,12 @@ import {
 import { BasicPools } from "./../providers/BasicPoolsProvider"
 import { BasicToken } from "./../providers/TokensProvider"
 import { BigNumber } from "@ethersproject/bignumber"
+import CHILD_GAUGE_ABI from "../constants/abis/childGauge.json"
```

Comment:
Could we move `CHILD_GAUGE_ABI` to `constants` folder because it is using on `useContract` as well?

Reply:
Hmm I don't want to move abis to a different place. Was that what you meant? Unless I'm misinterpreting your message?